src/AspNet.Security.OAuth.Patreon/PatreonAuthenticationHandler.cs

```diff
@@ -34,7 +35,19 @@ public class PatreonAuthenticationHandler : OAuthHandler<PatreonAuthenticationOp
             [NotNull] AuthenticationProperties properties,
             [NotNull] OAuthTokenResponse tokens)
         {
-            using var request = new HttpRequestMessage(HttpMethod.Get, Options.UserInformationEndpoint);
+            string endpoint = Options.UserInformationEndpoint;
+
+            if (Options.Fields?.Count > 0)
```

Comment:
Looks like these two collections have no setter and are always non-null, so the `?` is probably not necessary.

Reply:
Good spot! Will amend.